src/store/hooks.ts

```diff
@@ -0,0 +1,6 @@
+import { TypedUseSelectorHook, useDispatch, useSelector } from 'react-redux';
+import { RootState } from './rootReducer';
+import { AppDispatch } from 'store';
+
+export const useAppSelector: TypedUseSelectorHook<RootState> = useSelector;
+export const useAppDispatch: () => AppDispatch = useDispatch;
```

Comment:
"thunk" word is redundant, because every dispatch will be called with redux-thunk middleware :) you can replace it with 'App' or just leave `useCurrentWeatherDispatch`